### ISSUE TITLE
Radio buttons are not coded as such

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1625]
Several options on the settings page are presented as radio buttons, but they are implemented using ‘span’ elements instead of actual radio buttons. This concerns the following options:
- ‘Color theme’ in the ‘appearance’ section.
- ‘Document storage in CryptDrive’ in the
‘CryptDrive’ section.
- ‘Tag filter’ in the ‘Kanban’ section.
As a result, these elements are only announced as ‘clickable’ by screen readers, failing to convey their true function and state. To resolve this issue, the ‘span’ elements should be replaced with properly coded radio buttons (‘<input type=”radio”>’). Additionally, each radio button input should be associated with a corresponding label using the ‘for’ attribute.
